pytorch/requirements.txt

```
@@ -1,9 +1,9 @@
 mpi4py
 plotly
 pytorch-ignite
-pytorch-lightning>=1.6.0
+pytorch-lightning>=2.0
```

In my understanding, the pytorch lightning examples works with the latest version without changes, so I guess we can keep the support for v1.6 as well. Do we have difficulties to keep the the current lower bound?

```suggestion
pytorch-lightning>=1.6
```

---

As you said, there are no difficulties in keeping >=1.6 since that syntax logic will install whichever is the latest version.

Regarding explicitly setting a minimum at 2.0 in the requirements file, I would say this is more of a request as we move PL and Lightning Fabric into 2.0.

---

Thanks for your comment.

Taking your comments into account, I've reconsidered our approach. Up until now, we've only changed version constraints when code modifications required it.

On the other hand, the current CI only tests against the latest version as you mentioned, so we can't be certain that the lower version bounds are being accurately checked.

Therefore, rather than setting a lower limit, it might be more realistic to simply remove the version constraints and ensure functionality solely with the latest version.

What do you think of it?

---

That sounds good to me!